File: 2. letnik/C++/vaje test/testvaje/vaja3.cpp
/*
Z uporabo while zanke, zapiši program, ki bo shranjeval vnesena pozitivna števila v tabelo z imenom ManjsaOd50 toliko ?asa, dokler bo vsota vnesenih števil manjša od 4000.  
Shrani in sešteva naj le števila, ki so manjša od 50. 
?e je število ve?je ali enako 50 naj izpiše: "Število je preveliko.".
Na koncu naj program naj izpiše, z ustreznim komentarjem, izra?unano vsoto.
*/

#include <iostream>
#include <cmath>
using namespace std;

int main()
{
	unsigned int x;
	int v;
	
	
	
	while(v < 4000){
		cout<<"Vnesi število"<<endl;
			cin>>x;
			if(x >= 50)
			{
				cout<<"Število je preveliko"<<endl;
			};
			
			if(x < 50)
			{
				v += x;
			};		
	};
	return 0;
};

File: 2. letnik/Java/naloge/Naloge3_2/src/vaja_3.java
import java.util.Scanner;
//Napiši program, ki izpiše vsoto števil deljivih s 5 na intervalu med 2 in Y.
public class vaja_3 {

	public static void main(String[] args) {
		Scanner vhod = new Scanner (System.in);
		
		System.out.println("Vnesi število veèje od 3: ");
		int y = vhod.nextInt();
		int v = 0;
		
		for(int i = 3; i<=y; i++)
		{
			if(5 % i == 0)
			{
				v = v + i;
			}
		}
		System.out.print("Vsota števil deljivih s 5 je: "+v);
	}

}
